templates/vhost/locations/proxy.erb

```diff
@@ -1,3 +1,4 @@
+<% if @proxy -%>
     proxy_pass            <%= @proxy %>;
     proxy_read_timeout    <%= @proxy_read_timeout %>;
     proxy_connect_timeout <%= @proxy_connect_timeout %>;
```

Comment:
No problem. I saw that yesterday while browsing the code to make changes in my environment. I think I'll pass for the PR this time (mostly because I don't have much time, but also because this module is a bit too complex for me) and I realize that I'm already using the parameter in the global directive.

I must say, though that this module rocks and it's very nice to see how many people invest that much time into it.